### ISSUE TITLE
Ps2 slim scph-77004 6C says : Could not read the disc.

### ISSUE THREAD

**Issue by Civiani** [CTurt/FreeDVDBoot#207]
On my side as well
Ps2 slim scph-77004 6C
Cd player 2.00
Ps driver 2.00
Dvd player 3.11E

Tried the method from SamuraisDoJo on Platinum and TDK DVDS and it doesn't work! Even with Fun-tuna and Fortuna with up-to-date releases and FreeDVDBoot All Slims and 3.11 only, nothing works. DVD movies are playable but with BootDVD says cound not read the disc. Obviosly 7700x are problematic!!!

Tried everything mentioned as well on "Troubleshooting - please read if the above didn't work" and nothing works for me. 
Any other user with Ps2 slim scph-77004 6C that had the same issue?

**Comment by ghost** [comment]
> On my side as well
> Ps2 slim scph-77004 6C
> Cd player 2.00
> Ps driver 2.00
> Dvd player 3.11E
> 
> Tried the method from SamuraisDoJo on Platinum and TDK DVDS and it doesn't work! Even with Fun-tuna and Fortuna with up-to-date releases and FreeDVDBoot All Slims and 3.11 only, nothing works. DVD movies are playable but with BootDVD says cound not read the disc. Obviosly 7700x are problematic!!!
> 
> Tried everything mentioned as well on "Troubleshooting - please read if the above didn't work" and nothing works for me.
> Any other user with Ps2 slim scph-77004 6C that had the same issue?

Yep, i tried burning several discs at the lowest speed(1x) but it doesn't seem to work, i am saving my last clean DVD for a fixed version. i have the same model as yours, i used IMGBURN, my ps2 reads everything fine, i played a couple of games no problemo with it, movies play fine so it's definitely a problem of the exploit that doesn't play nice with this particular model. I looked at multiple threads and i noticed that the scph-77001 has issues with it too.

**Comment by J013k** [comment]
"All PS2 Slims - English language.iso" works for SCPH-77004 and for SCPH-77001:
https://playstationdev.wiki/ps2devwiki/index.php?title=FreeDVDBoot_Compatibility_List

I also have SCPH-77004 and everything is fine with this (ISO) image.
